testdata/go/v3/memcached-operator/monitoring/metrics/health_metrics.go

```diff
@@ -0,0 +1,17 @@
+package metrics
+
+import (
+	"github.com/example/memcached-operator/monitoring/metrics/util"
+)
+
+var healthMetrics = []util.Metric{
+	{
+		Name: "memcached_deployment_status",
```

Comment:
@avlitman, @machadovilaca, where this metric is used? Should we call `SetHealthStatus()` somewhere?

Reply:
It's not being currently used, we created this file as a suggestion on how we can split metrics from different domains into multiple files.